### ISSUE TITLE
Hello, I have some questions regarding the model's source code.

### ISSUE THREAD

**Issue by Epocherrr** [ChloeeGrace/LLaMA-Unidetector#4]
Dear Author,

I have a few questions I would like to ask:

(1)  In your paper, the input to Stage 2 is the feature maps generated in Stage 1. However, in the source code, the input to Stage 2 appears to be real-world .jpg images. Could you please clarify the reason for this difference?

(2)  The fine-tuning phase of Stage 2 in the paper utilizes two CLIP models: CLIP-ViT and CLIP-ConvNet. But  source code shows only one instance of CLIP-ViT is initialized for the LLM model. Where is the definition and usage of CLIP-ConvNet implemented?

(3)  The Stage 2 loss function described in the paper consists of two parts: a loss for feature matching and the TerraOV-LLM model loss. However, in the source code, the LLaMA_adapter uses a cross-entropy loss. Where are the two loss functions from the paper defined in the code?

These are the questions that came up while I was reading the source code. If I have misunderstood anything, I would greatly appreciate any corrections or clarifications.
